File: KarateKlub/KarateKlub.Api/Models/SaveziPregledVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace KarateKlub.Api.Models
{
    public class SaveziPregledVM
    {
        public class Row {
            public int id;
            public string naziv;
        }
        public List<Row> rows;
    }
}

File: Android aplikacija/KarateKlub/app/src/main/java/com/example/x/karateklub/area_trener/model/SaveziPregledVM.java
package com.example.x.karateklub.area_trener.model;

import java.io.Serializable;
import java.util.List;

public class SaveziPregledVM implements Serializable {
    public  static class Row implements  Serializable{
        public int id;
        public String naziv;

    }
    public List<Row> rows;
}
